File: deno/src/dictbiz_detail/dictbiz_detail.resolver.ts
import * as dictbiz_detailService from './dictbiz_detail.service.ts';

/**
 * 获取 codes 对应的业务字典
 */
export async function getDictbiz(
  codes: string[] = [ ],
) {
  return await dictbiz_detailService.getDictbiz(codes);
}

File: rust/src/src/base/dictbiz_detail/dictbiz_detail_resolver.rs
use color_eyre::eyre::Result;

use super::dictbiz_detail_model::GetDictbiz;
use super::dictbiz_detail_service;

pub async fn get_dictbiz(
  codes: Vec<String>,
) -> Result<Vec<Vec<GetDictbiz>>> {
  
  let data = dictbiz_detail_service::get_dictbiz(
    codes,
  ).await?;
  
  Ok(data)
}
